tests/install/.simplecov

```
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'simplecov'
+require 'simplecov-cobertura'
+
+SimpleCov.start do
+  # Output both HTML and Cobertura XML for Codecov
+  formatter SimpleCov::Formatter::MultiFormatter.new([
+    SimpleCov::Formatter::HTMLFormatter,
+    SimpleCov::Formatter::CoberturaFormatter
```

RuboCop (1.84.0)

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)